src/javascripts/services/travel_tracker_service.js

```diff
@@ -0,0 +1,42 @@
+function fetchUsername() {
+  const fetchApi = fetch('https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers');
+  fetchApi.then(response => {
+    return response.json();
+  }).then(data => {
+    var travArray = (data.travelers);
+
+    // Change username variable to loginInput
+    var loginInputBefore = document.getElementById('input-username');
+    // Downcase loginInput and chop
+    var loginInput = loginInputBefore.value.toLowerCase();
+    // First 9 letters saved to variable called loginName
+    var loginName = loginInput.substring(0, 8);
+    // Right side of split saved to variable called loginId
+    var loginId = loginInput.substring(8, 10);
+
+    // If (loginName === "traveler") then successful username
+    if (loginName === "traveler" || loginName === "agency") {
```

Comment:
We should be able to remove this whole `if` block, as we dont need to console log any of this.  We should do all of our "authorization" logic in the login_helper...and this file can be used exclusively for interacting with the external API